### ISSUE TITLE
_1password-gui is out of date on 20.09 (no longer functions as a result)

### ISSUE THREAD

**Issue by davidarmstronglewis** [NixOS/nixpkgs#106684]
###### Checklist

- [X] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name:
nixos._1password-gui (1password)
nixpkgs._1password-gui (1password)

current version: "0.8.4"
desired version: "0.9.7"

###### Notify maintainers

maintainers:
@danieldk
@timstott 

###### Note for maintainers

It seems that the packaged version on NixOS 20.09 no longer functions due to no longer being supported. This means it's essentially broken for users on 20.09.

I'm not sure the process to go about updating something in 20.09, but it seems like this could be viewed as a bug rather than simply being out of date as the software is unusable as a result of the older version.

```
INFO  2020-12-11T13:16:45.503 ThreadId(9) [op_app::operations::app_channel] App::new(<redacted>/<redacted>, Some(<redacted>))
(node:14896) UnhandledPromiseRejectionWarning: {"error":"AppExpired","humanReadableLabel":null}
(node:14896) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:14896) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

![Screenshot from 2020-12-11 13 16 53](https://user-images.githubusercontent.com/6754950/101955814-5a979380-3bb3-11eb-827b-5b65335e44cf.png)

It does seem the package was updated on master here: https://github.com/NixOS/nixpkgs/pull/106426

Please tag this issue in your PR.

**Comment by danieldk** [comment]
Thanks for reporting this! I run unstable, so I didn't notice this. I have made a PR to backport the latest release #106722. Unfortunately, it also requires that we add Electron 11 to 20.09. Let's see if the release maintainers think this is acceptable.

**Comment by davidarmstronglewis** [comment]
Happy to help!

**Comment by danieldk** [comment]
The updates were merged in the release-20.09 branch, so it should work again once the nixos-20.09 channel moves (nixos-20.09-small should already have the changes).

**Comment by davidarmstronglewis** [comment]
@danieldk, thank you so much for your help! I can confirm that after upgrading to the latest 20.09 revision that 1Password works again.

Closing this issue.
